vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -399,6 +487,7 @@ void Promotion::ClaimTokens(
       braveledger_helper::Security::BlindTokens(tokens);
 
   if (blinded_tokens.size() == 0) {
+    BLOG(ledger_, ledger::LogLevel::LOG_ERROR) << "Blided tokes are empty";
```

Comment:
Nit: Couple of misspellings in this log (blinded and tokens).

Reply:
done